libs/liquidity-provision/README.md

```diff
@@ -0,0 +1,7 @@
+# liquidity-provision
```

Comment:
chore (blocking): We already have a 'liquidity' lib which basically does the same thing. I don't think we need a new lib for this as they are too similar. Can you look into utilising whats in the already existing lib? Feel free to make changes there if required, but please check that the `liquidity/<marketid>` page on the console app still works :-)

Reply:
I've removed this lib and moved it into the `liquidity` lib. They don't share any code though - the biggest different is that `libs/liquidity/src/lib/liquidity-data-provider.ts` fetches data for only 1 market and I need data for all markets. I also need candles, and some other data + formatting since the dashboard is not _only_ about liquidity. However, there is another ticket to display liquidity providers for 1 market only (similar to `liquidity/<marketid>`) and at that point I'll be able to reuse code from this library.

Reply:
So ideally libs/liquidity would export hooks that can handle both use cases. Perhaps this is a single hook which takes an optional marketId argument or perhaps two separate hooks to keep it simpler.

If there is additional market data required for the page, I'd either split that up into a separate query or possibly use the market-data-provider in libs/market-list.

Reply:
Having a look at `libs/positions/src/lib/positions-data-providers.ts` was really helpful, thanks! :) I've created a second hook and I'm reusing `marketListProvider` & `marketsCandlesProvider` from `libs/market-list`.